### PR TITLE
bootstrap ci validation

### PR DESCRIPTION
**Pull request by sagadre**

### CODE REVIEW

File: open_lm/main.py [hunk 1]
```diff
@@ -189,46 +198,41 @@ def save_checkpoint(
...
-            previous_checkpoint = os.path.join(args.checkpoint_path, f"epoch_{completed_epoch - 1}.pt")
-            if os.path.exists(previous_checkpoint):
-                os.remove(previous_checkpoint)
-            previous_checkpoint = os.path.join(args.checkpoint_path, f"optimizer_{completed_epoch - 1}.pt")
-            if os.path.exists(previous_checkpoint):
-                os.remove(previous_checkpoint)
-            previous_checkpoint = os.path.join(args.checkpoint_path, f"stats_{completed_epoch - 1}.pt")
-            if os.path.exists(previous_checkpoint):
-                os.remove(previous_checkpoint)
+            for prefix in prefixes:
```
Comment (achalddave) [inline open_lm/main.py:232]: ❤️

**Comment by sagadre** [comment]
closes #34

**Review by Vaishaal — APPROVED** [review]

**Comment by Vaishaal** [comment]
LGTM

File: open_lm/train.py [hunk 2]
```diff
@@ -43,6 +43,43 @@ def update(self, val, n=1):
...
+                acc += self.sums[ii]
+                denom += self.weights[ii]
+
+            acc /= denom
+
+            estimates.append(acc)
+
+        half = (100 - interval) / 2
+
+        lower = np.percentile(estimates, half).item()
```
Comment (ludwigschmidt) [inline open_lm/train.py:77]: Do we need `.item()` here?

**Pull request merged**